### PR TITLE
ART-9021 get rid of auto in canonical_builders_from_upstream

### PR DESCRIPTION
**Pull request by Ximinhan**
canonical_builders_from_upstream supported mode to limit to [ True, False ] remove the unused lib release_schedule.py

### CODE REVIEW

File: ocp-build-data-validator/validator/json_schemas/assembly_group_config.schema.json [hunk 1]
```diff
@@ -229,7 +229,7 @@
     "urls-": {},
     "canonical_builders_from_upstream": {
       "description": "Assert our base images, or listen to upstream",
-      "enum": [true, false, "auto", "off", "no", "on", "yes"]
```
[the remark this answers is not in the document]
Comment (Ximinhan) [reply ocp-build-data-validator/validator/json_schemas/assembly_group_config.schema.json:232]: the on and off also need to be replaced with true and false, let's limit the value to bool true and false

File: doozer/tests/test_distgit/test_image_distgit/test_image_distgit.py [hunk 2]
```diff
@@ -627,23 +625,6 @@ def test_canonical_builders_enabled(self, release_schedule, mock_datetime):
         self.img_dg.runtime.group_config.canonical_builders_from_upstream = True
         self.assertEqual(False, self.img_dg._canonical_builders_enabled())
 
-        # canonical_builders_from_upstream = 'on' in image config (override)
```
[the remark this answers is not in the document]
Comment (Ximinhan) [reply doozer/tests/test_distgit/test_image_distgit/test_image_distgit.py:630]: the on and off also need to be replaced with true and false, pr updated to limit supported mode to [ True, False ]

File: ocp-build-data-validator/validator/json_schemas/assembly_group_config.schema.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (locriandev) [reply ocp-build-data-validator/validator/json_schemas/assembly_group_config.schema.json:232]: Ok, so maybe we can use a boolean type instead?

[the remark this answers is not in the document]
Comment (Ximinhan) [reply ocp-build-data-validator/validator/json_schemas/assembly_group_config.schema.json:232]: yeah, pr updated

**Review by locriandev — COMMENTED** [review]
/lgtm

**Pull request merged**